pkg/tfgen/generate.go

```diff
@@ -662,20 +662,15 @@ func (of *overlayFile) Name() string { return of.name }
 func (of *overlayFile) Doc() string  { return "" }
 func (of *overlayFile) Copy() bool   { return of.src != "" }
 
-func GenerateSchemaAndRenames(info tfbridge.ProviderInfo, sink diag.Sink) (pschema.PackageSpec, Renames, error) {
```

Comment:
I did not like this expansion of public API since we already have GenerateSchemaWithOptions that gives the caller access to renames if needed; this new API over-emphasizes Renames internals. So I inlined it.